src/storage.ts

```diff
@@ -218,6 +218,36 @@ const TOTAL_TIMEOUT_DEFAULT = 600;
  */
 const MAX_RETRY_DELAY_DEFAULT = 64;
 
+/**
+ * Returns true if the API request should be retried, given the error that was
+ * given the first time the request was attempted.
+ * @const
+ * @private
+ * @param {error} err - The API error to check if it is appropriate to retry.
+ * @return {boolean} True if the API request should be retried, false otherwise.
+ */
+const RETRYABLE_ERR_FN_DEFAULT = function (err?: ApiError) {
+  if (err) {
+    if ([408, 429, 500, 502, 503, 504].indexOf(err.code!) !== -1) {
+      return true;
+    }
+
+    if (err.errors) {
```

Comment:
Is there a situation where ```err !== undefined``` but ```err.errors === undefined``` (or really anything non-enumerable)?

Reply:
I dont believe so. If there were, the bug would have already surfaced [here](https://github.com/googleapis/nodejs-common/blob/56764dc58634d151f27b67869d38a7ef97708b6c/src/util.ts#L513)

Reply:
I think it would be safe to remove this ```if``` statement in that case.

Reply:
Sorry I misread your original question. It is possible for err.errors to be undefined. For example, in this test it is undefined. https://github.com/googleapis/nodejs-storage/blob/cec5e15780a0fb726247f4c281391157bdaa1cda/test/index.ts#L307

Reply:
I see it now: https://github.com/googleapis/nodejs-common/blob/4a1b1ab2bdedb92e4c5b102f04b005c252887c63/src/util.ts#L228. Thanks for clarifying!